appserver/admingui/core/src/main/resources/templates/baseLayout.xhtml

```diff
@@ -312,7 +312,7 @@
 
         var layout = new YAHOO.widget.Layout({
            units: [
```

Comment:
Top bar needed to be expanded to allow the restart required message to still show.